packages/origin-backend/src/pods/certificate/certificate.controller.ts

```diff
@@ -37,4 +47,92 @@ export class CertificateController {
     ): Promise<CertificationRequestOffChainData> {
         return this.certificationRequestRepository.findOne(id);
     }
+
+    @Get('/:id')
```

Comment:
Should use auth system to authenticate user and allow only users commitments to be served.